.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Please add ```test.md``` in the .gitignore file. This file should be ignored by git and is not present in your GitHub repository as stated in the project requirement.
The reason for ignoring using .gitignore is to avoid committing transient files from your working directory that aren't useful to other collaborators. 
You can read this [blog](https://www.bmc.com/blogs/gitignore/) to get an understanding about .gitignore and why we do ignore files on git.